firmware/targets/furi_hal_include/furi_hal_rtc.h

```diff
@@ -28,6 +28,9 @@ typedef enum {
     FuriHalRtcFlagDebug = (1 << 0),
     FuriHalRtcFlagFactoryReset = (1 << 1),
     FuriHalRtcFlagLock = (1 << 2),
+    FuriHalRtcFlagExecutePreUpdate = (1 << 3),
```

Comment:
Because functions setting and clearing these flags operate on single bits. We can introduce a separate 2-bit bitfield in reserved 16-bit area + additional API to change it, but is it really necessary?

Reply:
Using separate bits is telling that they are separate entities and can be used simultaneously which they not

Reply:
moved to separate bitfield